src/hooks/useWallet.ts

```diff
@@ -749,6 +753,41 @@ const useWallet = (): WalletState => {
     [],
   )
 
+  const getClientAllowance = useCallback(
```

Comment:
```suggestion
  const getClientContractAllowance = useCallback(
```

Otherwise we have `getClientAllowance` and `getAllowanceForClient` which sound almost identical, yet do 2 completely different things.

Reply:
`getAllowanceForClient` - old way, without client contract address, it calling Filecoin.StateVerifiedClientStatus function.
`getClientAllowance` - new one, for client.sol contract

so, ok, I will change the new function name so as not to be confused